Resources/config/parameters.yml

```diff
@@ -1,2 +1,3 @@
 parameters:
-    skuola_seo.twig_pagination_meta.extension.class: OpenSkuola\SeoBundle\Extension\Twig\PaginationMetaExtension
\ No newline at end of file
+    skuola_seo.twig_pagination_meta.extension.class: OpenSkuola\SeoBundle\Extension\Twig\PaginationMetaExtension
+    skuola_seo.bad_route_info_params: ["direction", "sort"]
```

Comment:
It's ok but if you want to follow the symfony conventions se this: http://symfony.com/doc/current/bundles/configuration.html